src/styles/Landing.module.css

```diff
@@ -0,0 +1,93 @@
+.wrapper {
+  display: flex;
+  width: 100%;
+  height: 100%;
+}
+
+.wrapper section {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.leftside-section {
+  background: linear-gradient(225deg, #005EAA 0%, #26418F 100%);
+}
+
+.logo-section-divider {
+  opacity: 0.25;
+}
+
+.login-content {
+  width: 22rem;
+}
+
+.login-btn {
+  font-size: 1rem;
```

Comment:
Don't buttons get a 1rem font size by default?

Reply:
Yes, I've made the updates based on the comments so far.